apps/api/src/app/events/usecases/parse-event-request/parse-event-request.usecase.ts

```diff
@@ -29,8 +29,16 @@ export class ParseEventRequest {
 
   async execute(command: ParseEventRequestCommand) {
     const transactionId = command.transactionId || uuidv4();
+    Logger.log('Starting Trigger');
 
     const mappedActor = command.actor ? this.mapTriggerRecipients.mapSubscriber(command.actor) : undefined;
+
+    if (mappedActor === undefined) {
+      Logger.warn('mappedActor is undefined');
```

Comment:
It's not a warning in this case, not having actor is a totally normal state of the system.